.github/actions/spelling/excludes.txt

```diff
@@ -0,0 +1,4 @@
+\.png$
+^LICENSE\.txt$
+^\.dockerignore$
+^\.github/
```

Comment:
Was this file just copied from somewhere? These files are not even present in the repo. Apart from the `.github` directory but why do you want to exclude it?